npm-debug.log

```diff
@@ -0,0 +1,102 @@
+0 info it worked if it ends with ok
```

Comment:
I don't think we want this file - could we remove it?

Reply:
sure, removing it... @tansaku